code/__DEFINES/mobs.dm

```diff
@@ -504,6 +504,7 @@ GLOBAL_LIST_INIT(xenoupgradetiers, list(XENO_UPGRADE_BASETYPE, XENO_UPGRADE_INVA
 #define CASTE_IS_BUILDER (1<<10) //whether we are classified as a builder caste
 #define CASTE_IS_A_MINION (1<<11) //That's a dumb ai
 #define CASTE_PLASMADRAIN_IMMUNE (1<<12)
+#define CASTE_IS_NOT_IN_LIST (1<<13) // xenos with this flag aren't registererd towards bioscan, towards minion rally, towards minimap icons
```

Comment:
Comment and name are wrong, this is only for minimap?
And why not show on minimap?

Reply:
Still open. And I don't like the name really